packages/perseus/src/widgets/interactive-graphs/locked-figures/locked-label.tsx

```diff
@@ -0,0 +1,40 @@
+import {font} from "@khanacademy/wonder-blocks-tokens";
+import * as React from "react";
+
+import {getDependencies} from "../../../dependencies";
+import {lockedFigureColors, type LockedLabelType} from "../../../perseus-types";
+import {pointToPixel} from "../graphs/use-transform";
+import useGraphConfig from "../reducer/use-graph-config";
+
+import type {GraphDimensions} from "../types";
+
+export default function LockedLabel(props: LockedLabelType) {
+    const {coord, text, color, size} = props;
+    const {range, width, height} = useGraphConfig();
+
+    const graphInfo: GraphDimensions = {
+        range,
+        width,
+        height,
+    };
+
+    const [x, y] = pointToPixel(coord, graphInfo);
```

Comment:
You can pass the result of `useGraphConfig()` directly to `pointToPixel`, and then you don't have to destructure and restructure the objects.

```suggestion
    const [x, y] = pointToPixel(coord, useGraphConfig());
```